lib/index.js

```diff
@@ -8,6 +8,7 @@ var Spawn = require("child_process").spawn
   , Ul = require("ul")
   , OneByOne = require("one-by-one")
   , Sliced = require("sliced")
+  , Q = require('q')
```

Comment:
Go ahead please :) I need to focus now on my main project I will test new package for sure :D